packages/cells/style/widget.css

```diff
@@ -127,6 +127,35 @@
   right: 0;
   top: 0;
   bottom: 0;
+  background-position-y: calc(
+    0.5 * var(--jp-content-heading-margin-top)
+  ) !important;
```

Comment:
I found it looks better to set it up as a function of the font-size (to have a better y alignment).

```suggestion
```